addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -2165,11 +2169,14 @@ stock EmitSoundToAllExcept(soundFlags, const String:sample[], entity=SOUND_FROM_
 	{
 		return;
 	}
-
-	EmitSound(clients, total, sample, entity, channel, level, flags, volume, pitch, speakerentity, origin, dir, updatePos, soundtime);
+	
+	for(int i=0; i<total; i++)
+	{
+		EmitSoundToClient(clients[i], sample, entity, channel, level, flags, volume, pitch, speakerentity, origin, dir, updatePos, soundtime);
```

Comment:
Any reason this was changed to EmitSoundToClient?

Reply:
Was testing with different functions, will revert that after I test